extensions/amp-access/0.1/access-expr-impl.js

```diff
@@ -175,7 +175,13 @@ parseError: function parseError(str, hash) {
     if (hash.recoverable) {
         this.trace(str);
     } else {
-        throw new Error(str);
+        function _parseError (msg, hash) {
+            this.message = msg;
+            this.hash = hash;
+        }
+        _parseError.prototype = Error;
+
+        throw new _parseError(str, hash);
```

Comment:
babel6 was erroring out on some label statements in the generated jison file, i tried upgrading jison and regenerated the files but it was still failing and had to add the "transform-remove-strict-mode" plugin to get it to work.